easybuild/easyconfigs/g/goolf/goolf-1.5.16.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "Toolchain"
+
+name = 'goolf'
+version = '1.5.16'
```

Comment:
compared to the existing `goolf/1.5.14`, GCC is bumped from 4.8.2 to 4.8.3 and OpenBLAS is bumped from 0.2.8 to 0.2.13

Hence, I would bump the toolchain version here a bit more to better reflect the different in versions, e.g. to `1.5.20`?